### PR TITLE
QA-970: Adding testTags for radiobutton and floating options elements

### PR DESCRIPTION
**Pull request by ifernandezdiaz**
## 📔 Objective

This work is a subtask of [QA-947](https://bitwarden.atlassian.net/browse/QA-947), a story created to group all the native app views that contain elements without Automation IDs.
NOTE: Not all the elements will require an AutomationID. We are adding the ones we currently need so we can reduce the flakiness on some critical Mobile e2e tests

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[QA-947]: https://bitwarden.atlassian.net/browse/QA-947?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/vault/VaultContent.kt [hunk 1]
```diff
@@ -56,7 +56,7 @@ fun VaultContent(
                     showDivider = true,
                     modifier = Modifier
                         .fillMaxWidth()
-                        .testTag("FolderFilter")
+                        .testTag("VerificationCodesFilter")
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/vault/VaultContent.kt:59]: 👍

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/appbar/action/BitwardenOverflowActionItem.kt [hunk 2]
```diff
@@ -65,23 +65,21 @@ fun BitwardenOverflowActionItem(
...
+                .testTag("FloatingOptionsContent")
                 .widthIn(
                     min = 112.dp,
                     max = 280.dp,
                 ),
             content = {
                 menuItemDataList.forEach { dropdownMenuItemData ->
                     DropdownMenuItem(
-                        modifier = Modifier.semantics {
-                            dropdownMenuItemData.testTag?.let {
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/appbar/action/BitwardenOverflowActionItem.kt:76]: Please remove the `testTag` property from the `OverflowMenuItemData` if it is no longer is needed

Comment (ifernandezdiaz) [reply app/src/main/java/com/x8bit/bitwarden/ui/platform/components/appbar/action/BitwardenOverflowActionItem.kt:76]: got it!

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/appbar/action/BitwardenOverflowActionItem.kt [hunk 3]
```diff
@@ -65,23 +65,21 @@ fun BitwardenOverflowActionItem(
...
-                                testTag = it
-                            }
-                        },
+                        modifier = Modifier.testTag("FloatingOptionsItem"),
                         colors = bitwardenMenuItemColors(),
                         text = {
                             Text(
                                 text = dropdownMenuItemData.text,
                                 style = BitwardenTheme.typography.bodyLarge,
+                                modifier = Modifier.testTag("FloatingOptionsItemName")
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/appbar/action/BitwardenOverflowActionItem.kt:82]: CI is failing on these trailing commas right now

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/dialog/row/BitwardenSelectionRow.kt [hunk 4]
```diff
@@ -56,6 +58,7 @@ fun BitwardenSelectionRow(
             text = text(),
             color = BitwardenTheme.colorScheme.text.primary,
             style = BitwardenTheme.typography.bodyLarge,
+            modifier = Modifier.testTag("AlertSelectionOptionName")
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/dialog/row/BitwardenSelectionRow.kt:61]: Missed this one too

**Comment by david-livefront** [comment]
There are still lint failures

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/listitem/BitwardenListItem.kt [hunk 5]
```diff
@@ -186,7 +186,7 @@ fun BitwardenListItem(
 data class SelectionItemData(
     val text: String,
     val onClick: () -> Unit,
-    val testTag: String? = null,
+    val testTag: String? = "AlertSelectionOption",
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/listitem/BitwardenListItem.kt:189]: Why can't we just apply this in the composable directly and remove this property just like you did with the `OverflowMenuItemData`

File: app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/vault/VaultEntryListItem.kt [hunk 6]
```diff
@@ -52,6 +52,7 @@ fun VaultEntryListItem(
                 SelectionItemData(
                     text = option.title(),
                     onClick = { onOverflowOptionClick(option) },
+                    testTag = "AlertSelectionOption",
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/vault/VaultEntryListItem.kt:55]: This is not needed

**Pull request merged**